src/libmongoc/doc/mongoc_oidc_credential_new_with_expires_in.rst

```diff
@@ -0,0 +1,38 @@
+:man_page: mongoc_oidc_credential_new
+
+mongoc_oidc_credential_new()
+============================
```

Comment:
```suggestion
mongoc_oidc_credential_new_with_expiry()
========================================
```

Reply:
Interesting, Sphinx does not appear to emit a warning for this mistake.